UPGRADING.md

```diff
@@ -124,7 +124,26 @@ for more information about these overrides.
 For more discussion of this, see [ADR 074](https://tinyurl.com/adr074), which
 lays out the reasoning for the changes as well as [RFC
 009](https://tinyurl.com/rfc009) for a discussion of the complexities of
-upgrading consensus parameters.
+upgrading consensus parameters.\
+
+### CLI Changes
+
+The functionality around resetting a node has been extended to make it safer. The CLI
+now has four `reset` subcommands: `blockchain`, `peers`, `unsafe-signer` and `unsafe-all`
+in replacement of the previous `unsafe-reset-all`.
+
+- `tendermint reset blockchain`: clears a node of all blocks, tendermint state, evidence,
+  and indexed transactions. NOTE: this won't affect application state which will need
+  to be separately reset. Please look into `tendermint rollback` in the event that you
+  need to rollback the last state due to application non-determinisim.
+- `tendermint reset peers`: clears the peer store which persist information on peers used
+  by the networking layer. This could be important when clearing stale addresses or switching
+  to using a defined set of peers.
```

Comment:
```suggestion
- `tendermint reset peers`: Clears the peer store, which persists information on peers used
  by the networking layer. This can be used to get rid of stale addresses or to switch
  to a predefined set of static peers.
```